src/components/productdisplayitem.main.jsx

```diff
@@ -31,6 +31,7 @@ import {
 import imgPlaceholder from '../images/img-placeholder.png';
 import ProductRecommendationsDisplayMain from './productrecommendations.main';
 import cortexFetch from '../utils/Cortex';
+import epConfig from '../ep.config.json';
```

Comment:
This is not necessary as it's already loaded below:
`const Config = require('Config');`

Reply:
Removed